runtime/lua/vim/lsp/callbacks.lua

```diff
@@ -62,6 +62,14 @@ M['textDocument/documentSymbol'] = function(_, _, result, _, bufnr)
   api.nvim_command("wincmd p")
 end
 
+M['workspace/symbol'] = function(_, _, result, _, bufnr)
+  if not result or vim.tbl_isempty(result) then return end
+
+  util.set_qflist(util.symbols_to_items(result, bufnr))
+  api.nvim_command("copen")
+  api.nvim_command("wincmd p")
+end
+
```

Comment:
```suggestion
local symbol_callback = function(_, _, result, _, bufnr)
  if not result or vim.tbl_isempty(result) then return end

  util.set_qflist(util.symbols_to_items(result, bufnr))
  api.nvim_command("copen")
  api.nvim_command("wincmd p")
end

M['textDocument/documentSymbol'] = symbol_callback
M['workspace/symbol'] = handle_symbol = symbol_callback
```

Reply:
> `M['workspace/symbol'] = handle_symbol = symbol_callback`

Good idea! But is the `handle_symbol` in there intended?

Reply:
Sorry, it's typo. I've fixed comnents.